client/state/activity-log/rewind-status/reducer.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Internal dependencies
+ */
+import { rewindStatusSchema } from './schema';
+import {
+	REWIND_STATUS_ERROR,
+	REWIND_STATUS_UPDATE,
+} from 'state/action-types';
+import {
+	createReducer,
+	keyedReducer,
+} from 'state/utils';
+
+export const rewindStatus = createReducer( {}, {
+	[ REWIND_STATUS_ERROR ]: keyedReducer( 'siteId', () => null ),
+	[ REWIND_STATUS_UPDATE ]: keyedReducer( 'siteId', ( state, { status } ) => status ),
+}, rewindStatusSchema );
+
+export const rewindStatusErrors = createReducer( {}, {
```

Comment:
I think I prefer to keep the names singular, except when the per-site content will be a list. So `logItems` makes sense because we expect a list of items for a site. But with errors we only expect one value per-site, so I would expect the name to be `rewindStatusError`. We can take it as given that there will be multiple sites.